README.md

```diff
@@ -89,7 +98,7 @@ const vsac = require('cql-exec-vsac');
 // Set up the code service, loading from the cache if it exists
 const codeService = new vsac.CodeService('/path/to/vsac_cache', true);
 // Ensure value sets in the library and its dependencies, downloading any missing value sets
-codeService.ensureValueSetsInLibrary(library, true, 'myUMLSUserName', 'myUMLSPassword')
+codeService.ensureValueSetsinLibraryWithApiKey(library, true, 'myUmlsApiKey')
```

Comment:
In this code snippet and a few other places in this file, `ensureValueSetsinLibraryWithApiKey` should technically be `ensureValueSetsInLibraryWithApiKey` and `ensureValueSetsinLibrary` should be `ensureValueSetsInLibrary`. The one seems the most important to update since it's in the code example, but while you're changing it, it might be nice to fix the few other places as well.

```suggestion
codeService.ensureValueSetsInLibraryWithApiKey(library, true, 'myUmlsApiKey')
```

Reply:
Good catch! This and others fixed in 0c993b5.